### PR TITLE
docs(devserver): add documentation for liveReload option

### PR DESCRIPTION
**Pull request by EslamHiko**
add docs for the feature in this pr: https://github.com/webpack/webpack-dev-server/pull/1889

### CODE REVIEW

File: src/content/configuration/dev-server.md [hunk 1]
```diff
@@ -659,6 +659,29 @@ T> [`watchOptions`](#devserver-watchoptions-) will have no effect when used with
 
 T> If you use the CLI, make sure __inline mode__ is disabled.
 
+## `devServer.liveReload`
+
+`boolean: true`
+
+When `devServer.liveReload` is disabled, the dev-server will never reload/refresh the page.
```
Comment (wizardofhogwarts) [inline src/content/configuration/dev-server.md]: `never` appears to be a bit awkward.

Comment (EslamHiko) [reply src/content/configuration/dev-server.md]: @wizardofhogwarts i updated the description with more detailed one, can you please check it ? thanks.

File: src/content/api/cli.md [hunk 2]
```diff
@@ -299,6 +300,7 @@ Parameter         | Explanation                              | Usage
 `--define`        | Define any free variable, see [shimming](/guides/shimming) | `--define process.env.NODE_ENV="'development'"`
 `--hot`           | Enables [Hot Module Replacement](/concepts/hot-module-replacement) | `--hot=true`
 `--labeled-modules` | Enables labeled modules [Uses LabeledModulesPlugin] |
+`--live-reload`           | Enables page live reloading | `--live-reload=true`
```
Comment (EugeneHlushko) [inline src/content/api/cli.md]:
```suggestion
`--live-reload`           | Enables live reloading | `--live-reload=true`
```

File: src/content/configuration/dev-server.md [hunk 3]
```diff
@@ -696,6 +696,29 @@ T> [`watchOptions`](#devserver-watchoptions-) will have no effect when used with
 
 T> If you use the CLI, make sure __inline mode__ is disabled.
 
+## `devServer.liveReload`
+
+`boolean: true`
+
+When `devServer.liveReload` is enabled, the dev-server will reload/refresh the page when file changes if [`devServer.hot`](#devserverhot) option is disabled or [`devServer.watchContentBase`](#devserverwatchcontentbase) is enabled.
```
Comment (EugeneHlushko) [inline src/content/configuration/dev-server.md]:
```suggestion
By default, the dev-server will reload/refresh the page when file changes are detected. [`devServer.hot`](#devserverhot) option must be disabled or [`devServer.watchContentBase`](#devserverwatchcontentbase) option must be enabled in order for `liveReload` to take effect. Disable `devServer.liveReload` by setting it to `false`:
```

File: src/content/configuration/dev-server.md [hunk 4]
```diff
@@ -696,6 +696,29 @@ T> [`watchOptions`](#devserver-watchoptions-) will have no effect when used with
...
+
+__webpack.config.js__
+
+```javascript
+module.exports = {
+  //...
+  devServer: {
+    liveReload: true
```
Comment (EugeneHlushko) [inline src/content/configuration/dev-server.md]:
```suggestion
    liveReload: false
```

File: src/content/configuration/dev-server.md [hunk 5]
```diff
@@ -696,6 +696,29 @@ T> [`watchOptions`](#devserver-watchoptions-) will have no effect when used with
...
+  }
+};
+```
+
+Usage via the CLI
+
+```bash
+webpack-dev-server --live-reload
```
Comment (EugeneHlushko) [inline src/content/configuration/dev-server.md]: Isn't this default? Why we need to pass it

**Comment by EslamHiko** [comment]
@EugeneHlushko Done 😄.

**Comment by EugeneHlushko** [comment]
Thanks!

**Pull request merged**